boards/common/atmega/board.c

```diff
@@ -23,10 +23,25 @@
 #include "irq.h"
 #include "periph/gpio.h"
 
+#ifndef CPU_ATMEGA_CLK_SCALE_INIT
+#define CPU_ATMEGA_CLK_SCALE_INIT    CPU_ATMEGA_CLK_SCALE_DIV1
```

Comment:
This should go into `./boards/common/arduino-atmega/include/periph_conf.h` preferably under the `CLOCK_CORECLOCK` definition and with a small bit of doxygen.

Can be done without the `#ifndef` construction but I don't mind keeping it.

Reply:
Problem though: not all boards using boards/common/atmega are Arduinos.

Reply:
I think we can solve that problem in a follow up PR. Discussed a bit offline with @bergzand and maybe found a solution.